scrapy/extensions/robotstxtparser.py

```diff
@@ -0,0 +1,130 @@
+import sys
+import logging
+from six.moves.urllib_robotparser import RobotFileParser
+
+from scrapy.utils.python import to_native_str, to_unicode
+
+logger = logging.getLogger(__name__)
+
+class PythonRobotParser():
+    def __init__(self, content, crawler):
+        self.crawler = crawler
```

Comment:
is this variable used?

Reply:
No, I have removed that variable in the follow-up commit.